src/test/rpc/Book_test.cpp

```diff
@@ -800,6 +800,460 @@ class Book_test : public beast::unit_test::suite
             books)[jss::status] == "success");
     }
 
+    void testTrackOffers(void)
+    {
+        using namespace jtx;
+        Env env(*this);
+        Account gw {"gw"};
+        auto wsc = makeWSClient(env.app().config());
+        env.fund(XRP(20000), "alice", "bob", gw);
+        env.close();
+        auto USD = gw["USD"];
+
+        Json::Value books;
+        {
+            books[jss::books] = Json::arrayValue;
+            {
+                auto &j = books[jss::books].append(Json::objectValue);
+                j[jss::snapshot] = true;
+                j[jss::taker_gets][jss::currency] = "XRP";
+                j[jss::taker_pays][jss::currency] = "USD";
+                j[jss::taker_pays][jss::issuer] = gw.human();
+            }
+
+            auto jv = wsc->invoke("subscribe", books);
+            BEAST_EXPECT(jv[jss::status] == "success");
+            BEAST_EXPECT(jv[jss::result].isMember(jss::offers) &&
+                jv[jss::result][jss::offers].size() == 0);
+            BEAST_EXPECT(! jv[jss::result].isMember(jss::asks));
+            BEAST_EXPECT(! jv[jss::result].isMember(jss::bids));
+        }
+
+        env(rate(gw, 1.1));
+        env.close();
+        env.trust( USD(1000), "alice" );
+        env.trust( USD(1000), "bob" );
+        env(pay(gw, "alice", USD(100)));
+        env(pay(gw, "bob",   USD(50)));
+        env(offer("alice", drops(4000), USD(10)));
```

Comment:
It looks like orderbook-test.js uses XRP.

        // get 4000/XRP pay 10/USD : offer pays 10 USD for 4000 XRP
        var tx = $.remote.transaction('OfferCreate', {
          account: 'alice',
          taker_pays: '4000',
          taker_gets: '10/USD/mtgox'
        });